back/src/forms/elastic.ts

```diff
@@ -172,7 +243,7 @@ function toBsdElastic(form: FullForm): BsdElastic {
  * Index all BSDs from the forms table.
  */
 export async function indexAllForms(
-  idx: string,
+  idx: string | null | undefined,
```

Comment:
À priori `indexAllForms` est toujours appelé avec un index défini et n'est pas préparé à recevoir `null | undefined` (la fonction `indexBsds` attend une string).

```suggestion
  idx: string,
```